flixel/input/mouse/FlxMouseEventManager.hx

```diff
@@ -443,22 +443,30 @@ class FlxMouseEventManager extends FlxBasic
 		for (camera in Register.object.cameras)
 		{
 			#if FLX_MOUSE
-			_point = FlxG.mouse.getWorldPosition(camera, _point);
-			
-			if (checkOverlapWithPoint(Register, _point, camera))
+			_point = FlxG.mouse.getScreenPosition(camera, _point);
+			if (camera.containsPoint(_point))
 			{
-				return true;
+				_point = FlxG.mouse.getWorldPosition(camera, _point);
+				
+				if (checkOverlapWithPoint(Register, _point, camera))
+				{
+					return true;
+				}
 			}
 			#end
 			
 			#if FLX_TOUCH
 			for (touch in FlxG.touches.list)
 			{
-				_point = touch.getWorldPosition(camera, _point);
-				
-				if (checkOverlapWithPoint(Register, _point, camera))
+				_point = touch.getScreenPosition(camera, _point);
```

Comment:
so this line should be changes as well to 
`_point = touch.getPositionInCameraView(camera, _point);`